### PR TITLE
added profile

### PR DESCRIPTION
**Pull request by MichaelBoitmann**
thank you for adding my profile

### CODE REVIEW

File: Submissions/@michaelboitmann.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Michael Boitmann",
+    "img": "https://avatars.githubusercontent.com/u/55775047?s=400&u=9125d841568874d0f4a54b158e9f17dd004b3921&v=4",
+    "email":"michael_boitmann@yahoo.com",
+    "links": {
+      "website": "#",
```
Comment (ProsperousHeart) [inline Submissions/@michaelboitmann.json:6]: please remove the #

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]
Please remove the # for website, the @ in front of your file, and delete the file for nazmul that is not yours. Otherwise it will change this person's data.

**Comment by MichaelBoitmann** [comment]
> Please remove the # for website, the @ in front of your file, and delete the file for nazmul that is not yours. Otherwise it will change this person's data.

Thanks. Sure i will the firstv2 items but for nazmul, that is the problem i have. After clone process it shows 2 names under naznul which are @nazmul and @Nazmul without ne doing anything in thise files. Tried to reclone it on 4 different folder but showing the same issue and I don't know why and how to fix it. Kindly delete those pull request i created as i created another one without Nazmul. Thanks

File: Submissions/@michaelboitmann.json (hunk 1 above)
Comment (MichaelBoitmann) [reply Submissions/@michaelboitmann.json]: done. thank you

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]
you still seem to have an extra change outside of the file you're trying to submit. Please undo changes in that file as outlined in your [Files Changed](https://github.com/zero-to-mastery/ZtM-Job-Board/pull/2713/files)

**Pull request closed without merging**